### PR TITLE
Update main page

### PR DESCRIPTION
**Pull request by sudodoki**
Updated main page - added coaches / toggable year navigation, sponsor section

### CODE REVIEW

**Comment by sudodoki** [comment]
@Kottans/owners considering this done in terms of style. Can preview it at http://mashatimmpa.github.io/3_kottans.
//cc @MashaTimmpa 
There's some work to do regarding content, fixing names, adding photos & social links.

**Comment by suchov** [comment]
http://prntscr.com/6idiru

File: public/styles/home.css [hunk 1]
```diff
@@ -172,26 +371,6 @@ h3 {
   width: 65%;
 }
 
-#contact form:invalid button {
```
Comment (sudodoki) [inline public/styles/home.css]: Still need to return this.

File: public/styles/home.css [hunk 2]
```diff
@@ -221,11 +400,10 @@ h3 {
   padding: 8px 45px;
   background: #fff;
   border: none;
-  cursor: pointer;
```
Comment (sudodoki) [inline public/styles/home.css]: need to return this

File: public/styles/home.css [hunk 3]
```diff
@@ -283,7 +461,7 @@ h3 {
   -webkit-filter: grayscale(100%);
 }
 
-#slides .disabled::after {
```
Comment (sudodoki) [inline public/styles/home.css]: and this

File: public/styles/home.css [hunk 4]
```diff
@@ -319,9 +497,7 @@ footer p {
   color: #fff;
 }
 
-.language {
```
Comment (sudodoki) [inline public/styles/home.css]: need to return this.

**Comment by sudodoki** [comment]
Considering this done. Commented bunch of bios - you could find those [intact](https://github.com/Kottans/kottans.github.io/blob/update-main/_config.yml#L14). Might want to return those (& truncate the one by @suchov).

**Comment by sudodoki** [comment]
This PR should be closing #28, #16, #13, #5 & might lead to closing #33.

**Comment by sudodoki** [comment]
Todo: fix vendor/js for new plugin

**Comment by suchov** [comment]
So, can we close it?

**Comment by sudodoki** [comment]
@suchov if by close you mean merge, then yep, everything is done.

**Pull request merged**